src/contexts/useSdkConfigurationService.tsx

```diff
@@ -0,0 +1,62 @@
+import { getSdkConfiguration } from '~utils/onfidoApi'
+import { useContext, useEffect, useState } from 'preact/compat'
+import { h, ComponentChildren, createContext, Fragment } from 'preact'
+import { SdkConfiguration } from '~types/api'
+import deepmerge from 'deepmerge'
+
+type SdkConfigurationServiceProviderProps = {
+  children: ComponentChildren
+  url?: string
+  token?: string
+  fallback?: ComponentChildren
+}
+
+const defaultConfiguration: SdkConfiguration = {}
+const envConfiguration = process.env.SDK_CONFIGURATION as SdkConfiguration
```

Comment:
Why is this an env variable and it's an empty object? Is this to allow us to pass the configurations we want at development time?

Reply:
These could only be changed at build time, though. Do we want that?

Reply:
* Default configuration is the fallback when we can't reach the server.
* envConfiguration, configuration in webpack, is the specific configuration per env, to override the server configuration (Ex: We may want to disable something specific in E2E).

Reply:
I see. Wouldn't it make more sense to expose a function to update the configuration at runtime for a specific env so you don't need to rebuild?